jwst/assign_wcs/util.py

```diff
@@ -769,7 +769,7 @@ def _create_grism_bbox(input_model, mmag_extract=99.0,
                     pts = np.array([[ymin, xmin], [ymax, xmax]])
                     ll = np.array([0, 0])
                     ur = np.array([input_model.meta.subarray.ysize, input_model.meta.subarray.xsize])
-                    inidx = np.all(np.logical_and(ll <= pts, pts <= ur), axis=1)
+                    inidx = np.all(np.logical_and(ll < pts, pts < (ur-1)), axis=1)
```

Comment:
Reviewers: still not convinced I have this 100% correct - the goal being to disallow bboxes with width 1 on either edge. If xmin and xmax are both 0 or 2047 (in the case of xsize=2048), does this do what I'd like?